unison-src/transcripts/api-getDefinition.output.md

```diff
@@ -104,14 +104,14 @@ GET /api/non-project-code/getDefinition?names=names.x&relativeTo=nested
                 ]
             ],
             "termNames": [
-                "names.x"
+                "nested.names.x"
```

Comment:
This is a small behavioural change; I talked it over with Simon.

We now always include the term name qualified to the project root rather than the perspective.